src/stylesheets/components/_list-styles.scss

```diff
@@ -108,6 +108,45 @@ dl, dt, dd {
   }
 }
 
+/*--- List Events ---*/
+
+.list--events {
+  @include list;
+  margin-bottom: $base-spacing-unit-medium;
+
+  > li {
+    padding: 0;
+    margin-bottom: 0;
+
+    @include media($desktop) {
+      @include flex(33.333%, 66.666%);
+      align-items: flex-start;
+      border-top: 1px solid $grey-3;
+      border-bottom: 1px solid $grey-3;
+      margin-bottom: -1px;
+    }
+
+    &:first-child {
+      @include media(0, $desktop) {
+        border-top: 0;
+      }
+    }
+
+    > :nth-child(2) {
```

Comment:
This would only be either an `<ol>` or a `<ul>` let's make it more specific and rather than nesting three levels deep change it to:

```
list--events > li {
  > ol,
  > ul {
    // insert here
  }
```